vllm_ascend/models/deepseek_v2.py

```diff
@@ -315,15 +315,26 @@ def __init__(
         self.enable_multistream_moe = \
             ascend_config.torchair_graph_config.enable_multistream_moe and \
             self.torchair_graph_enabled
+        self.enable_super_kernel = self.enable_multistream_moe and self.tp_size == 1
+        self.params_dtype = torch.get_default_dtype()
```

Comment:
use the following suggestion to avoid duplicated code?
```suggestion
        self.params_dtype = torch.float32 if self.enable_super_kernel else torch.get_default_dtype()
```

Reply:
Thank you for your suggestion.